handlers/lib/config.js

```diff
@@ -1,17 +1,18 @@
 // if the 3rd field is blank, the variable must be set
 const configString = `
-  SFTP_RETRY_QUEUE_NAME            push,retry      
-  SFTP_TARGET_DIR                  push,retry      
+  SFTP_RETRY_QUEUE_NAME            push,retry
+  SFTP_TARGET_DIR                  push,retry
   SFTP_PUSH_TIMEOUT_SECONDS        push,retry      20
   SFTP_SOURCE_S3_REGEXP_STRIP      push,retry      optional
-  SFTP_HOST                        all             
-  SFTP_PORT                        all             
-  SFTP_USER                        all             
-  SFTP_PRIVATE_KEY                 all             
+  SFTP_HOST                        all
+  SFTP_PORT                        all
+  SFTP_USER                        all
+  SFTP_PASSWORD                    all             optional
+  SFTP_PRIVATE_KEY                 all             optional
```

Comment:
I've set both of these variables as `optional`, even though one of them typically has to be set. New code in `envVar` should handle this. Let me know if this approach should be changed.